src/Npgsql/NpgsqlBuffer.cs

```diff
@@ -618,5 +618,71 @@ struct BitConverterUnion
         }
 
         #endregion
+
+        #region Postgis
+        //  Postgis returns bytes in ndr representation.
+        internal int PostgisReadInt32()
```

Comment:
Do I understand correctly that PostGIS uses little endian everywhere? If so can you please rename the methods to `ReadInt32LSB`?